### PR TITLE
bootc build VM task

### PR DESCRIPTION
**Pull request by ralphbean**
This is a duplicate of https://github.com/konflux-ci/build-definitions/pull/1059. I'm picking that up since stuart is on PTO.

I tried to address @mmorhun's first round of feedback there, here.

I'll close that one so we can focus on this one (although https://github.com/konflux-ci/build-definitions/pull/1042 is still valid).

### CODE REVIEW

**Comment by ralphbean** [comment]
I think this is blocked on https://github.com/konflux-ci/build-definitions/pull/1137.

**Comment by ralphbean** [comment]
Submitted https://github.com/enterprise-contract/hacks/pull/102

File: task/build-vm-image/0.1/build-vm-image.yaml [hunk 1]
```diff
@@ -0,0 +1,263 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/pipelines.minVersion: 0.12.1
+    tekton.dev/tags: image-build, appstudio, hacbs
```
Comment (mmorhun) [inline task/build-vm-image/0.1/build-vm-image.yaml:6]: Do we still need `hacbs`?

**Review by mmorhun — APPROVED** [review]

File: task/build-vm-image/0.1/build-vm-image.yaml (hunk 1 above)
Comment (ralphbean) [reply task/build-vm-image/0.1/build-vm-image.yaml:6]: Almost certainly not. I can drop it and add konflux, or we can do a mass `sed` PR later to update it across the board?

Comment (ralphbean) [reply task/build-vm-image/0.1/build-vm-image.yaml:6]:
```
❯ git grep tekton.dev/tags | grep hacbs | wc -l
30
```
Let's do it in a separate PR and get them all at once. :)

**Comment by ralphbean** [comment]
Thank you for the review @mmorhun.

**Pull request merged**